mrjones2014/dash.nvim
# Add 'css' to default keywords list for CSS preprocessors like Sass, Less, Stylus

Resolves: #122

## How to Test

1. Open a `.sass` file, a `.scss` file, a `.less` file, and a `.styl` file
2. For each one, run `:Dash`
3. It should filter Dash by both its own docset and the CSS docset
